first-analysis-steps/davinci-grid.md

```diff
@@ -47,7 +49,7 @@ with the option files given in `j.application.options` using a
 backend called `Dirac`, which is "the grid". Instead of specifying the
 files to process as part of the options file you have now to tell the
 `Job` about it. This allows `ganga` to split your job up,
-processing different files simultaneously.
+processing different files simultaneously. Note that data will be accessed using its path in the bookeeping `bkPath`. In order to speed-up, only the first 2 elements (files) of `data` will be accessed.
```

Comment:
* Two 'k's in bookkeeping.
* "In order to speed up _our job_, ... will be accessed; we don't need to look at much data here."